### DISCUSSION TITLE
IPBlacklist Policy is not working and needs a guidance for testing the IP Blacklist policy to check if it is working or not

### DISCUSSION THREAD

**Discussion by mansighule1** [apiman/apiman#2519] [General]
Hi 

I added the IP blacklist policy to the API, It's generating the response that Policy added successfully and added my system's IP in the blacklist's list.
I hit the generated endpoint(using postman) from my system's IP and then also I'm able to access that endpoint and not getting any response that access from this IP is restricted.

Want a detail guidance over that how to test the IP blacklist/whitelist policy and check if the policy is applied to the particular endpoint
And would be thankful for the help.

**Comment by msavy** [comment]
Please open a bug issue, this is not enough data for us to help you 

https://github.com/apiman/apiman/issues/new?assignees=&labels=T%3A+Bug%2CS%3A+Triage&template=bug.yml
